sdk/logs/src/main/java/io/opentelemetry/sdk/logs/SdkLoggerProviderBuilder.java

```diff
@@ -108,7 +109,10 @@ public SdkLoggerProviderBuilder setClock(Clock clock) {
   }
 
   /**
-   * Set the logger configurator, which computes {@link LoggerConfig} for each {@link
+   * This method is experimental so not public. You may reflectively call it using {@link
+   * SdkLoggerProviderUtil#setLoggerConfigurator(SdkLoggerProviderBuilder, ScopeConfigurator)}
+   *
+   * <p>Set the logger configurator, which computes {@link LoggerConfig} for each {@link
    * InstrumentationScopeInfo}.
```

Comment:
These are flipped. By convention, the first sentence of javadoc is a short summary of method. This new text provides supplementary information, which is important but secondary to the summary sentence.